src/test/java/com/nhantran/chapter10/CreateAccountTest.java

```diff
@@ -1,61 +1,58 @@
 package com.nhantran.chapter10;
 
 import com.nhantran.base.TestBase;
-import com.nhantran.pages.*;
-import com.nhantran.utils.DriverManager;
+import com.nhantran.enums.RailwayTabs;
+import com.nhantran.models.User;
+import com.nhantran.pages.HomePage;
+import com.nhantran.pages.MailboxPage;
+import com.nhantran.pages.RegisterPage;
+import com.nhantran.pages.RegistrationConfirmationPage;
+import com.nhantran.utils.Constants;
 import com.nhantran.utils.SeleniumActions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-import java.util.Set;
-
 public class CreateAccountTest extends TestBase {
 
     HomePage homePage = new HomePage();
     RegisterPage registerPage = new RegisterPage();
     MailboxPage mailboxPage = new MailboxPage();
     RegistrationConfirmationPage registrationConfirmationPage = new RegistrationConfirmationPage();
-
-    public CreateAccountTest() {
-        super();
-    }
+    private User alreadyRegistedUser = new User(Constants.VALID_USERNAME, "0987654321", "0987654321", "000000000000");
+    private User emptyPasswordPidUser = new User(Constants.VALID_USERNAME, null, null, null);
+    private User newUser;
 
     @Test(description = "User can't create account with an already in-use email")
     public void TC007_FailToCreateAccountWithEmailAlreadyUsed() {
-        homePage.clickTab("Register");
-        registerPage.register(validUsername, validPassword, validPassword, "0909090909090909");
+        homePage.clickTab(RailwayTabs.REGISTER);
+        registerPage.register(alreadyRegistedUser);
         Assert.assertEquals(registerPage.getErrorMessageAboveRegisterForm(), "This email address is already in use.", "The error message does not match");
```

Comment:
should store these error messages into another file (e/x: constants) to easy to manage